test_fixtures/simple_tagger_without_span_f1/experiment.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
I don't think we need to add another fixture. This case should already be covered by https://github.com/allenai/allennlp/blob/master/test_fixtures/simple_tagger/experiment.json, right?

Reply:
It was intended to check the case `calculate_span_f1` is `false`. (In `experiment.json`, `calculate_span_f1` would be `None`, I think.)

But I also think it is too much, then removed in a6be2ae.

Reply:
Gotcha! Yea, I don't think that really needs a test. Anyway, thank you! Looks good!